### DISCUSSION TITLE
fsi6x  APM32 failed to flash OpenTX

### DISCUSSION THREAD

**Discussion by Flyingpiglets** [OpenI6X/opentx#344] [Q&A]
In the case of the APM32 MCU,Flashing with DFUProgrammer failed.
After the flashing is complete, the boot card cannot exit in the failsafe.
Ask for your help! Thank!

**Comment by Ultraexpert** [comment]
> In the case of the APM32 MCU,Flashing with DFUProgrammer failed.
> 
> After the flashing is complete, the boot card cannot exit in the failsafe.
> 
> Ask for your help! Thank!

Did you get it sorted? I can help with the APM32

**Reply by Flyingpiglets** [reply]
No, it still failed

**Reply by Ultraexpert** [reply]
That's odd I have APM32 and 1.8 version it works without hassle

**Comment by ajjjjjjjj** [comment]
Is it the same on 1.8.0?

**Reply by Flyingpiglets** [reply]
yes

**Reply by ajjjjjjjj** [reply]
Is everything working fine, receiver binds on original software?

**Reply by Flyingpiglets** [reply]
The first update 1.8.0 can be used normally, and the subsequent shutdown and power on again find that it is stuck on the fault page and cannot exit. I tried updating to 1.7.0 and the result is still the same.

**Reply by Flyingpiglets** [reply]
The remote control calibration was not carried out after the first update, and I don't know if it is the reason for it.
